acceptance/tests/apply/augeas/services.rb

```diff
@@ -0,0 +1,61 @@
+test_name "Augeas services file" do
+
+  tag 'risk:medium'
+
+  confine :except, :platform => 'windows'
+  confine :except, :platform => 'osx'
```

Comment:
From https://github.com/puppetlabs/pe_acceptance_tests/commit/1d25a1f676e5b4fcef5d685b83bcf7590d127424, "* Augeas does not parse services file on OSX".